README

```diff
@@ -108,6 +108,10 @@ Mandriva:
 	Mandriva has integrated fio into their package repository, so installing
 	on that distro should be as easy as typing ``urpmi fio``.
 
+Arch Linux:
+        Arch Linux package is provided under community.
```

Comment:
This was more to emphase that the package under the "sub-repository" named community.

Eventually we can skip the community thing because it could eventually be understandable as the repository maintained by standard user (AUR).